src/chunkserver/datastore/chunkserver_datastore.h

```diff
@@ -187,6 +188,27 @@ class CSDataStore {
                                   SequenceNum sn,
                                   char * buf,
                                   off_t offset,
+                                  size_t length) {
+        butil::IOPortal iop;
+        CSErrorCode rc = ReadChunk(id, sn, &iop, offset, length);
+        iop.copy_to(buf, length, 0);
```

Comment:
the third parameter is 0 by default, maybe you should not specify it explicitly

Reply:
But it's harmless